libs/github.ts

```diff
@@ -9,7 +9,7 @@ export function restGithub<T = any>(path: string, preview = 'hellcat') {
     authorization: `token ${pickGithubToken()}`,
     accept: `application/vnd.github.${preview}-preview+json`
   }
-  const prefixUrl = 'https://api.github.com/'
+  const prefixUrl = process.env.GITHUB_URL || 'https://api.github.com/'
```

Comment:
I think `GITHUB_API` might be a better naming for this, and `GITHUB_API_GRAPHQL` for the other one 😄